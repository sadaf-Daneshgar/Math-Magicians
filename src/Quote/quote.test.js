import React from 'react';
import renderer from 'react-test-renderer';
import ShowQuote from './quote';

test('ShowQuote component snapshot', () => {
  const tree = renderer.create(<ShowQuote />).toJSON();
  expect(tree).toMatchSnapshot();
});
