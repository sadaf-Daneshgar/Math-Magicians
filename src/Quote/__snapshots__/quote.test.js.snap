// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowQuote component snapshot 1`] = `
<div
  className="quote-container"
>
  <div
    className="title-quote"
  >
    Amazing Quotes ‚≠êÔ∏è
  </div>
  <p
    className="quote"
  />
  <h2
    className="author"
  >
    üë§ By:
  </h2>
  <p
    className="catagory"
  >
    Type of Quote:
  </p>
</div>
`;
